sources/osgGA/OrbitManipulatorStandardMouseKeyboardController.js

```diff
@@ -73,6 +73,13 @@ OrbitManipulatorStandardMouseKeyboardController.prototype = {
                 } else {
                     this.setMode( OrbitManipulatorEnums.ROTATE );
                 }
+            }
```

Comment:
I checked in Sketchfab with Chrome, Firefox, Opera and Safari and only Firefox triggers this elseIf block. :)